Makefile

```diff
@@ -1,4 +1,5 @@
-VERSION ?= 0.16.2
+# TODO(cbuto): Pull from git tag to avoid needing to bump this prior to releasing
```

Comment:
Since we run this inside Action , I think this will be helpful . May be `$(shell git describe --tags --abbrev=0)` ? And then , we should run the action when every new tag pushed ? 🤔